pkg/v1/resource/certconfig/desired.go

```diff
@@ -199,6 +203,37 @@ func newCalicoCertConfig(clusterConfig cluster.Config, cert certs.Cert, projectN
 	}
 }
 
+func newClusterOperatorAPICertConfig(clusterConfig cluster.Config, cert certs.Cert, projectName string) *v1alpha1.CertConfig {
```

Comment:
Looks like only `string(cert)` is used in `newClusterOperatorAPICertConfig`, could we pass this string instead of the whole `certs.Cert`?

Reply:
What is wrong about that? Why losing type safety?

Reply:
We can keep type safety, but checking on the required string only. We are passing a whole struct value when only one of the members is needed.

Reply:
I don't follow. `Certs` isn't a struct: https://github.com/giantswarm/certs/blob/9383ad5288f22c6fa4ab5412d37fecefdc81d37c/k8s.go#L24

Reply:
I am who wasn't following, you are totally right, scratch all the above :)

Reply:
Good to check but going with this since its correct.